android/src/main/kotlin/com/adyen/checkout/flutter/utils/Constants.kt

```diff
@@ -9,5 +9,6 @@ class Constants {
         const val GOOGLE_PAY_ADVANCED_COMPONENT_KEY = "GOOGLE_PAY_ADVANCED_COMPONENT"
         const val GOOGLE_PAY_SESSION_REQUEST_CODE = 486350
         const val GOOGLE_PAY_ADVANCED_REQUEST_CODE = 486351
+        const val SDK_PAYMENT_CANCELED_IDENTIFIER = "Payment canceled"
```

Comment:
If this is not a string that will be displayed or logged anywhere, consider changing to to something that looks like a constant, for example "PAYMENT_CANCELED"

Reply:
This string is only used for identifying if the Google Pay sheet is closed by the user. The native SDK provides a ComponentException with "Payment canceled" as the error message. This is used to map the ComponentException to a "PaymentCancelledByUser" outcome. 

This is where the ComponentException gets provided in the native SDK: https://github.com/Adyen/adyen-android/blob/52e05ce148ac454683a5fa92d20900d8e45c9e96/googlepay/src/main/java/com/adyen/checkout/googlepay/internal/ui/DefaultGooglePayDelegate.kt#L156